modules/cnf-about-topology-aware-lifecycle-manager-policies.adoc

```diff
@@ -18,4 +18,4 @@ For policies that update an Operator subscription with manual approval, {cgu-ope
 
 For more information about managed policies, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/governance/index#policy-overview[Policy Overview] in the {rh-rhacm} documentation.
 
-For more information about the `PolicyGenTemplate` CRD, see the "About the PolicyGenTemplate" section in "Deploying distributed units at scale in a disconnected environment".
+For more information about the `PolicyGenTemplate` CRD, see the "About the PolicyGenTemplate CRD" section in "Configuring managed clusters with policies and PolicyGenTemplate resources".
```

Comment:
Two nits: This change should also apply to [4.11](https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.html) and [4.10](https://docs.openshift.com/container-platform/4.10/scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.html), I believe. 

Also, consider making this an _Additional reference_ in the assembly so that you can add a link. Might make it easier for the user.

Reply:
Addressed